.github/drivers/mock-cache.sh

```diff
@@ -12,7 +12,7 @@ for module_version_dir in /tmp/kobuild-tmp/versions-src/*; do
     mkdir -p "/tmp/kernel-modules/$module_version/"
 
     # If the bucket doesn't exist, we are building a new version of drivers
-    if ! gsutil ls "gs://mauro-drivers-test/drivers/${module_version}/" 1>&2 2> /dev/null; then
+    if ! gsutil ls "gs://mauro-drivers-test/drivers/${module_version}/" > /dev/null; then
```

Comment:
am I correct in thinking it was the order of the old redirections that was wrong? don't we now get stderr written to the logs? does any get written at all?

Reply:
Probably, though `redirect stdout to stderr and stderr to /dev/null` looks correct to me, I need to brush up on bash redirections. Nothing get print to the logs, unless we get an error which... Sounds kinda useful?